engine/crates/engine-v2/src/operation/bind/selection_set.rs

```diff
@@ -151,21 +157,24 @@ impl<'schema, 'p, 'binder> SelectionSetBinder<'schema, 'p, 'binder> {
         &mut self,
         ty: SelectionSetType,
         selection_set: &'p Positioned<engine_parser::types::SelectionSet>,
+        parent_directive_value_ids: (&[QueryInputValueId], &[QueryInputValueId]),
```

Comment:
would be simpler to read with something like:

```rust
struct ExecutableDirectives {
  skip_input_values_ids: Vec<QueryInputValueId>,
  include_input_value_ids: Vec<QueryInputValueId>
}

impl ExecutableDirectives {
  fn extend(&mut self, parent_directives: &ExecutableDirectives) { ... }
}
```

Reply:
Done